README.md

```
@@ -1,9 +1,41 @@
+
+### Install Dependencies
+
+Before launching the dashboard, install dependencies:
+
+```
+pip install -r requirements.txt
+```
+
+### Launch Dashboard
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)